packages/wasm/src/services/credential/service.ts

```diff
@@ -199,7 +199,7 @@ class CredentialService {
   }) {
     const searchParams = new URL(uri).searchParams;
     const params = new URLSearchParams(searchParams);
-    const credentialOfferEncoded = params.get('credential_offer');
+    const credentialOfferEncoded = params.get('credential_offer') || params.get('credential_offer_uri');
```

Comment:
can we test that this works somehow?

Reply:
yep, let me add proper tests here

Reply:
this logic is incorrect, `credential_offer_uri` is not an encoded credential offer. the lines immediately below it will fail. its a uri you have to request to get the offer JSON

Reply:
@maycon-mello why was this marked as resolved? i still see the same code/issues as the comment above

Reply:
I didn't see that comment I guess, let me handled that one too.